### PR TITLE
Fix Brazilian Portuguese translation

### PR DESCRIPTION
**Pull request by carlos-henrique-dev**
Fix some words that were in pt-PT to pt-BR and finishes the translation from English to Brazilian Portuguese

### CODE REVIEW

File: kofta/public/locales/pt-BR/translation.json [hunk 1]
```diff
@@ -1,211 +1,214 @@
...
+    "loadMore": "Carregar mais",
+    "loading": "Carregando...",
+    "noUsersFound": "Nenhum usuário encontrado",
+    "ok": "Ok",
+    "yes": "Sim",
+    "no": "Não",
+    "cancel": "Cancelar",
+    "save": "Salvar",
+    "edit": "Edit",
+    "delete": "Delete",
```
Comment (fsmiamoto) [inline kofta/public/locales/pt-BR/translation.json:13]: Shouldn't it be `Deletar` here and `Editar` above?

File: kofta/public/locales/pt-BR/translation.json [hunk 2]
```diff
@@ -1,211 +1,214 @@
...
+      "startRoom": "Iniciar sala",
+      "modal": {
+        "needsFuture": "Precisa ser uma data no futuro",
+        "roomName": "Nome da sala",
+        "minLength": "O nome precisa ter pelo menos 2 caracteres"
+      }
+    },
+    "roomChat": {
+      "title": "Chat",
+      "emotesSoon": "[emotes soon]",
```
Comment (fsmiamoto) [inline kofta/public/locales/pt-BR/translation.json:202]: Maybe translate this one?

File: kofta/public/locales/pt-BR/translation.json [hunk 3]
```diff
@@ -1,211 +1,214 @@
...
+      "featureText_4": "Código Aberto",
+      "featureText_5": "Conversa de Texto",
+      "featureText_6": "Desenvolvido por Doge",
+      "loginGithub": "Entrar com GitHub",
+      "loginTwitter": "Entrar com Twitter",
+      "createTestUser": "Criar usuário teste"
+    },
+    "myProfile": {
+      "logout": "Sair",
+      "probablyLoading": "Prossívelmente carregando...",
```
Comment (fsmiamoto) [inline kofta/public/locales/pt-BR/translation.json]: Here we have a typo
```suggestion
      "probablyLoading": "Possivelmente carregando...",
```

**Review by fsmiamoto — COMMENTED** [review]
Some things that I noticed, but nice one.

File: kofta/public/locales/pt-BR/translation.json (hunk 2 above)
Comment (carlos-henrique-dev) [reply kofta/public/locales/pt-BR/translation.json:202]: I don't think "chat" needs to be translated and didn't really know what he meant with "emotes soon" so I left it out. 
What do you think it should be?

Comment (fsmiamoto) [reply kofta/public/locales/pt-BR/translation.json:202]: Agreed on the `Chat`, I've meant to point out only the `emotes soon`.
We could leave it as is or use something like `emotes logo`.

**Comment by benawad** [comment]
switch this to `staging` branch too

**Comment by nadirabbas** [comment]
please pull from `staging` instead of `new-design`

**Pull request closed without merging**